doc/setup.md

```diff
@@ -9,7 +9,11 @@ Click the icon in the upper left to view the table of contents.
 
 resetti has only been tested with [MultiMC](https://multimc.org) and forks such
 as [PrismLauncher](https://prismlauncher.org). No functionality is guaranteed
-when using the vanilla launcher or any other launcher.
+when using the vanilla launcher or any other launcher. 
+
+> **Warning**
+> resetti does not currently, work with launchers installed using Flatpak.
```

Comment:
Extra comma after currently

Reply:
After this fix it should be good to merge.

Reply:
Me and my commas smh